index.html

```diff
@@ -157,12 +157,12 @@ <h2 class="card__content--heading">Multi-Post Stories</h2>
 <section class="aboutsec">
         <div class="about">
   <div class="about__content">
-    <h2 class="aheading">About <br> Myself</h2>
-    <p class="ades">
+    <h1 class="aheading">About <br> Myself</h1>
+    <h2 class="ades">
       Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-    </p>
-    <p class="aget">LET'S CONNECT</p>
```

Comment:
- [ ] Instead of using h2 here blush Please use `p` because this is the paragraph of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1: